services/store.go

```diff
@@ -976,6 +976,7 @@ func decodeTransfer(height uint32, info *event.ExecuteNotify, evt *event.NotifyE
 			}
 			res := big.NewInt(0).Mul(big.NewInt(amount), big.NewInt(constants.GWei))
 			res.Add(res, big.NewInt(value))
+			res.Mul(res, big.NewInt(1e9))
```

Comment:
when  len(elems) == 5 Note that the transfer amount contains decimal places, which is the processing after magnifying the accuracy.   like https://github.com/ontio/ontology-rosetta/blob/master/services/store.go#L977